README.md

```diff
@@ -1 +1,171 @@
-# Final-Capstone-Back-End
\ No newline at end of file
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 DriveShare App  <a name="about-project"></a>
+
+DriveShare is a Full-stack web application. that makes easier to Add, reserve and delete cars.
+
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+<summary>Ruby on rails</summary>
+  <ul>
+    <li><a href="https://guides.rubyonrails.org/index.html">Ruby on rails</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- Rails Api
+- Add and remove cars
+- reserve car
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+### Link To frontend Repo <a name="key-features"></a>
+[Link](https://github.com/Zohra-Neda/DriveShare-Front-End)
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+### Link to Kanban Board <a name="key-features"></a>
+
+- It's a four-member team.
+- Check here [Kanban Board](https://github.com/Zohra-Neda/DriveShare-Back-End/projects/1)
+- Check here the initial state of the [Kanban Board Screenshot](https://user-images.githubusercontent.com/76215321/278043042-bd79c58f-351d-4d94-a4b6-f5809559c909.png)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+> This project requires you to have ruby and rails on your Local Environment.
+> You also need to have Node installed on your local Environment
+
+### Setup
+
+Clone this repository to your desired folder:
+
+```sh
+  git clone https://github.com/Zohra-Neda/DriveShare-back-End.git
+  cd DriveShare-back-End
+```
+
+
+### Install
+
+Install this project with:
+
+```sh
+  gem install rails
+```
+
```

Comment:
- [ ] Kindly, include in your instructions the required db commands that your users will need to run your backend app: ``rails db:create, rails db:migrate, rails db:seed``. 👍